pkgs/development/tools/gauge/wrapper.nix

```diff
@@ -0,0 +1,56 @@
+{ gauge-unwrapped
+, gauge
+, makeWrapper
+, stdenvNoCC
+, lib
+, xorg
+, gaugePlugins
+, plugins ? []
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "gauge-wrapped";
+  inherit (gauge-unwrapped) version;
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out{bin,/share/gauge/{plugins,config}}
+    export GAUGE_HOME="$out/share/gauge"
```

Comment:
If we need to remove logs afterwards, maybe it is easier to only copy over, what we do need?

Reply:
done!